projects/storefrontlib/src/lib/cart/components/add-to-cart/add-to-cart.component.spec.ts

```diff
@@ -26,6 +21,12 @@ class MockCartService {
       '1234': { entryNumber: 0, product: { code: productCode } }
     });
   }
+  getLoaded(): Observable<boolean> {
```

Comment:
it's better to change it to the property.

Reply:
Done.